test/e2e/testnet.go

```diff
@@ -178,6 +240,12 @@ func (t *Testnet) Cleanup() {
 			log.Err(err).Msg(fmt.Sprintf("node %s failed to cleanup", node.Name))
 		}
 	}
+	if t.txSimNode.Instance != nil {
+		err := t.txSimNode.Instance.Destroy()
+		if err != nil {
+			log.Err(err).Msg(fmt.Sprintf("txsim %s failed to cleanup", t.txSimNode.Name))
+		}
+	}
```

Comment:
I think we need to stop the instance first and then destroy it

Reply:
Good point, added this https://github.com/celestiaorg/celestia-app/pull/3194/commits/131e021dd7840190bce67a3743f562812d7bb905